src/views/TopologyDisplay.vue

```diff
@@ -0,0 +1,340 @@
+              <v-spacer/>
+            </v-toolbar>
+            <div style="height: 100%; width: 100%; display: flex; flex-direction: row">
+              <ComponentsPanel :displays="displays" :series="timeSeriesStore"/>
+            </div>
+          </pane>
+        </splitpanes>
+      </pane>
+    </splitpanes>
+  </div>
+</template>
+<script lang="ts">
+
+import {Component, Mixins, Prop, Watch} from "vue-property-decorator";
+import TimeSeriesMixin from "@/mixins/TimeSeriesMixin";
+import PiRequestsMixin from "@/mixins/PiRequestsMixin";
+import {ColumnItem} from "@/components/ColumnItem";
+import {
+  ActionRequest,
+  ActionsResponse, Location,
+  PiWebserviceProvider,
+  TopologyActionFilter,
+  TopologyNode
+} from "@deltares/fews-pi-requests";
+import {DisplayConfig, DisplayType} from "@/lib/Layout/DisplayConfig";
+import ColumnMenu from "@/components/ColumnMenu.vue";
+import TreeMenu from "@/components/TreeMenu.vue";
+import ComponentsPanel from "@/components/Layout/ComponentsPanel.vue";
+import {TopologyDisplayTab} from "@/lib/TopologyDisplay/types";
+import {TopologyNodeResponse} from "@deltares/fews-pi-requests/lib/types/response/topology";
+import ExplorerComponent from "@/components/explorer/ExplorerComponent.vue";
+import {Pane, Splitpanes} from 'splitpanes'
+import 'splitpanes/dist/splitpanes.css'
+import {timeSeriesDisplayToChartConfig} from "@/lib/ChartConfig/timeSeriesDisplayToChartConfig";
+import {namespace} from "vuex-class";
+import {fetchTimeSeriesDisplaysAndRequests} from "@/lib/TopologyDisplay/timeseriesdisplay";
+
+const sytemTimeModule = namespace('systemTime')
+
+@Component({
+  components: {
+    ExplorerComponent,
+    ColumnMenu,
+    TreeMenu,
+    ComponentsPanel,
+    Splitpanes,
+    Pane
+  }
+})
+
+export default class TopologyDisplay extends Mixins(TimeSeriesMixin, PiRequestsMixin) {
+  @Prop({default: '', type: String})
+  nodeId!: string
+
+  static TIME_SERIES_DIALOG_PANEL: string = "time series dialog";
+
+  @sytemTimeModule.State('startTime')
+  startTime!: Date
+  @sytemTimeModule.State('endTime')
+  endTime!: Date
+
+  viewMode = 0
+  active: string[] = []
+  open: string[] = []
+  columnItems: ColumnItem[] = []
+  tabs: TopologyDisplayTab[] = []
+  baseUrl!: string
+  webServiceProvider: PiWebserviceProvider = {} as PiWebserviceProvider;
+  urlTopologyNodeMap: Map<string, string> = new Map<string, string>()
+  displays: DisplayConfig[] = []
+  selectedTab = null;
+  allDisplays: DisplayConfig[][] = []
+  requests: ActionRequest[][] = [];
+  plots: string[] = [];
+  selectedItem: number = -1
+  wmsLayerId: string | undefined = ''
+  filterIds: string[] | undefined = []
+  topologyNode: TopologyNode | undefined = undefined
+  showMap = true
+  showTimeSeries = false
+  topologyMap: Map<string, TopologyNode> = new Map<string, TopologyNode>();
+
+
+  created(): void {
+    this.baseUrl = this.$config.get('VUE_APP_FEWS_WEBSERVICES_URL')
+  }
+
+  async mounted(): Promise<void> {
```

Comment:
Het gaat nu nog niet goed als je direct naar een URL in de topologie-hiërarchie navigeert